iroha-cli/interactive/impl/interactive_transaction_cli.cpp

```diff
@@ -0,0 +1,443 @@
+      std::vector<std::string> notes = {
+          "Account ID", "Public key (must be one of the signatories):"};
+      auto params = parseParams(line, ASSIGN_M_KEY, notes);
+      if (not params.has_value()) {
+        return nullptr;
+      }
+      auto account_id = params.value()[0];
+      auto key = params.value()[1];
+      iroha::ed25519::pubkey_t pubkey;
+      iroha::hexstringToArray(key, pubkey);
+      return generator_.generateAssignMasterKey(account_id, pubkey);
+    }
+
+    std::shared_ptr<iroha::model::Command>
+    InteractiveTransactionCli::parseCreateAccount(std::string line) {
+      std::vector<std::string> notes = {"Account name", "Domain id",
+                                        "Public Key"};
+      auto params = parseParams(line, CREATE_ACC, notes);
+      if (not params.has_value()) {
+        return nullptr;
+      }
+      auto account_id = params.value()[0];
+      auto domain_id = params.value()[1];
+      auto key = params.value()[2];
+      iroha::ed25519::pubkey_t pubkey;
+      iroha::hexstringToArray(key, pubkey);
+      return generator_.generateCreateAccount(account_id, domain_id, pubkey);
+    }
+
+    std::shared_ptr<iroha::model::Command>
+    InteractiveTransactionCli::parseCreateDomain(std::string line) {
+      std::vector<std::string> notes = {"Full domain id"};
+      auto params = parseParams(line, CREATE_DOMAIN, notes);
+      if (not params.has_value()) {
+        return nullptr;
+      }
+      auto domain_id = params.value()[0];
+      return generator_.generateCreateDomain(domain_id);
+    }
+
+    std::shared_ptr<iroha::model::Command>
+    InteractiveTransactionCli::parseCreateAsset(std::string line) {
+      std::vector<std::string> notes = {"Full domain id"};
+      auto params = parseParams(line, CREATE_ASSET, notes);
+      if (not params.has_value()) {
+        return nullptr;
+      }
+      auto asset_name = params.value()[0];
+      auto domain_id = params.value()[1];
+      auto val = parser::toInt(params.value()[2]);
+      if (not val.has_value()) {
+        // error
+        return nullptr;
+      }
+      return generator_.generateCreateAsset(asset_name, domain_id, val.value());
+    }
+
+    std::shared_ptr<iroha::model::Command>
+    InteractiveTransactionCli::parseRemoveSignatory(std::string line) {
+      std::vector<std::string> notes = {"Account ID", "Public key to remove"};
+      auto params = parseParams(line, ADD_SIGN, notes);
+      if (not params.has_value()) {
+        return nullptr;
+      }
+      auto account_id = params.value()[0];
+      auto key = params.value()[1];
+      iroha::ed25519::pubkey_t pubkey;
+      iroha::hexstringToArray(key, pubkey);
+      return generator_.generateRemoveSignatory(account_id, pubkey);
+    }
+
+    std::shared_ptr<iroha::model::Command>
+    InteractiveTransactionCli::parseSetPermissions(std::string line) {
+      // TODO: implement when change permission model
```

Comment:
With this reason what the purpose for now to do this method? May be better remove it?